bears/js/JSHintBear.py

```diff
@@ -6,6 +7,20 @@
 from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
 
 
+def bool_or_str(value):
```

Comment:
This is invalid now.
This was me getting confused about `value`.
I thought `value` was the Setting's value - but it's the Setting itself.